packages/clerk-js/src/ui/components/ApiKeys/ApiKeysTable.tsx

```diff
@@ -163,7 +180,10 @@ export const ApiKeysTable = ({
                     </Text>
                   </Flex>
                 </Td>
-                <Td>
+                <Td
+                  elementDescriptor={descriptors.apiKeysTableCellLastUsed}
+                  elementId={descriptors.apiKeysTableCellLastUsed.setId(apiKey.id)}
```

Comment:
I don't think there is strong value in added elementIds that map the the api key ids in elements and creates a bit more classes that becomes confusing. 
![Screenshot 2025-06-17 at 1 21 24 PM](https://github.com/user-attachments/assets/9dff639f-889d-4ff5-a2cb-5b2704e019c5)

Reply:
Now that you mention it, I can see the problem